contentPages/models.py

```diff
@@ -30,6 +30,7 @@ def get_resource_type_choices(cls):
                 resource_type_value = resource_type.lower().replace(' ', '_')
                 resource_types.append((resource_type_value, resource_type))
         except:
+
```

Comment:
No need for action but try and leave files like this out of PRs. It only confuses what the scope of the ticket is.